### DISCUSSION TITLE
为什么装了JDK 21和22之后虽然没有指定适合的Java，但是游戏还是各种崩溃

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#2405] [提问] [answered]
如题，而且有些报错虽然是Java的问题，但是却报去跟Java不相关的地方了，指定了java有些问题甚至就莫名其妙的没了
另外声明：如果要我传log可能得等一周左右，期间可以自己先试着

**Comment by iPodtouch6557** [comment]
其中可能有一部分是我mod装太多的问题，~毕竟一个原版生存就能让我装上三四十个辅助mod~
但是有些问题是指定Java或者重新启动一遍就能好

**Comment by XiaoLLiu617** [comment]
题目不清晰，可能崩溃原因就是没有指定适合的Java，建议详细描述或自行解决

**Comment by XiaoLLiu617** [comment]
> 如题，而且有些报错虽然是Java的问题，但是却报去跟Java不相关的地方了，指定了java有些问题甚至就莫名其妙的没了 另外声明：如果要我传log可能得等一周左右，期间可以自己先试着

报错出现错误是正常的（毕竟是机器）‘’指定了java有些问题甚至就莫名其妙的没了‘’这句话有问题，不易理解

**Comment by iPodtouch6557** [comment, marked as answer]
玄幻问题，大抵是我mod装多了……不玩就没问题了，我确实是这样做的（
~我都忘了还有这个了~
